lib/grunt.js

```diff
@@ -69,6 +69,12 @@ module.exports = function(grunt) {
 
   });
 
+  grunt.registerTask('wrapcodepoints', function() {
```

Comment:
This could probably use a comment describing why we wrap code points after the fact instead of doing it inline in the SASS